MPChartLib/src/main/java/com/github/mikephil/charting/interfaces/datasets/IDataSet.java

```diff
@@ -285,6 +285,13 @@ public interface IDataSet<T extends Entry> {
      */
     int getColor();
 
+    /**
+     * Returns the Gradient color model
+     *
+     * @return
+     */
+    GradientColor getGradientColor();
```

Comment:
**Suggestion**: Adding the getGradientColor method to the IDataSet interface increases the flexibility of the library. Ensure that all implementations of IDataSet properly support this method, returning meaningful gradient information or a default if not applicable. <br> **Code Suggestion**: 
 ```
 # Implement default handling of getGradientColor in datasets that do not support gradients, possibly returning null or a default GradientColor object.
 ```